lib/logstash/outputs/influxdb.rb

```diff
@@ -128,7 +128,7 @@ def receive(event)
     event_hash['name'] = event.sprintf(@series)
     sprintf_points = Hash[@data_points.map {|k,v| [event.sprintf(k), event.sprintf(v)]}]
     if sprintf_points.has_key?('time')
-      @logger.error("Cannot override value of time without 'allow_override_time'. Using event timestamp") unless @allow_override_time
+      @logger.error("Cannot override value of time without 'allow_time_override'. Using event timestamp") unless @allow_override_time
```

Comment:
This settings is probably a lost in translation... 

We migrated the plugin from logstash-contrib and this option wasn't used https://github.com/logstash-plugins/logstash-output-influxdb/commit/ccf8cc15b42bbbc434760ef09bce782e691e1fc4